src/components/translator/Translator.tsx

```diff
@@ -243,11 +243,16 @@ const WithTgtLang = ({
 const Translator = ({ mode: initialMode }: { mode?: Mode }): React.ReactElement => {
   const mode: Mode = initialMode || Mode.Text;
 
+  const textUrlParam = 'q';
```

Comment:
This shouldn't be repeated twice. It should be defined in exactly one place.

Reply:
I want to confirm if the suggestion is to define it globally, or if there's an alternative approach you recommend for maintaining it in a single place?

Reply:
You can define it one component (ideally the parent component) and export the constant for use in other files.